modules/bun.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, ... }:
+
+let
+  bun = pkgs.replitPackages.bun;
+
+  extensions = [".js" ".ts"];
+in
+
+{
+  name = "Bun Tools";
+  version = "1.0";
+
```

Comment:
I think we should add bun to the packages list so it is available on the commandline.

Reply:
fixed in latest commit